.github/workflows/stale.yml

```diff
@@ -13,6 +13,5 @@ jobs:
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           stale-issue-label: "Status: Stale"
-          stale-issue-message: "This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days"
+          stale-issue-message: "This issue is stale because it has been open 45 days with no activity."
           days-before-stale: 45
-          days-before-close: 7
```

Comment:
I don't think we want to remove this - looks like the default value is 7: https://github.com/actions/stale#days-before-close